tests/specs/tram-spec.js

```diff
@@ -128,8 +121,8 @@ const tests = (Tram) => describe('Tram', () => {
         .addRoute('/good', successPage)
         .addRoute('/bad', errorPage)
 
-      expect(app.toString('/good')).toEqual(stringify(successPage()))
-      expect(app.toString('/bad')).toEqual(stringify(errorPage()))
+      expect(app.toString('/good')).toEqual(successPage().outerHTML)
+      expect(app.toString('/bad')).toEqual(errorPage().outerHTML)
```

Comment:
Why do you not check the 404 route here as well?

Reply:
the test here is that `.addRoute` returns the Tram One object, and can be called after itself. 

The default 404 route is already tested here: 
https://github.com/Tram-One/tram-one/blob/c0c1323455e83ff70ac550a303adabb57df3e6aa/tests/specs/tram-spec.js#L29-L34

I guess it might be useful to check that a route isn't overwritten by the addition of other routes, but that would be logic to test in `rlite`